File: judge/src/tested/languages/java/templates/Encode.java
import java.util.*;
import java.io.*;

public class Encode {

    public static void main(String[] args) throws Exception {
        PrintWriter writer = new PrintWriter(System.out);
        % for statement in statements:
            Values.send(writer, <%include file="statement.mako" args="statement=statement"/>);
            writer.write("\n");
        % endfor
        writer.close();
    }

}

File: judge/src/tested/languages/kotlin/templates/Encode.kt
import java.io.PrintWriter

class Encode {
    companion object {
        @JvmStatic
        fun main(args : Array<String> = emptyArray()) {
            val writer = PrintWriter(System.out)
            % for statement in statements:
                valuesSend(writer, <%include file="statement.mako" args="statement=statement"/>)
                writer.write("\n")
            % endfor
            writer.close()
        }
    }

}
